test/Benchmarks/src/Startup/Startup.enso

```diff
@@ -35,20 +45,31 @@ type Data
 collect_benches = Bench.build builder->
     options = Bench.options . set_warmup (Bench.phase_conf 2 5) . set_measure (Bench.phase_conf 3 5)
 
-
     data =
         Data.Value enso_bin (find_sibling "Empty_World.enso") (find_sibling "Hello_World.enso") (find_sibling "Import_World.enso")
 
-    builder.group "Startup" options group_builder->
-        group_builder.specify "empty_startup" data.bench_empty
-        group_builder.specify "hello_world_startup" data.bench_hello
-        group_builder.specify "import_world_startup" data.bench_import
+    [["Native_Image_Startup", False], ["JVM_Startup", True]].each \entry->
```

Comment:
- It would be good to **not rename** the already executed benchmarks 
- otherwise the history is going to disconnect
- does that imply to keep "Startup" name for JVM launch?
- and introduce "Native Startup" for the new _native image_ launches?